src/main/resources/static/js/angular-parish-file/src/app/family/family.component.html

```diff
@@ -0,0 +1,92 @@
+<div id="test">
+  <a class="button"id="new-family-button" href="#popup1">New family</a>
+
+<div id="popup1" class="overlay">
+  <div class="popup">
+    <div id="new-family-form">
+      <label>Last name:
+        <input #surname [(ngModel)]="newFamily.surname"/>
+      </label>
+      <label>Address:
+        <input #address [(ngModel)]="newFamily.address"/>
+      </label>
+      <label>Phone number:
+        <input #phoneNumber [(ngModel)]="newFamily.phoneNumber"/>
+      </label>
+      <label>Last pastoral visit:
+        <input type="date" #lastPastoralVisit [(ngModel)]="newFamily.lastPastoralVisit"/>
+      </label>
+      <a class="button close window-close-button" href="#" (click)="save(); surname.value=''; address.value='';
```

Comment:
set empty values in save() method and if this is button to save so why there is close class name? Maybe it should be save.

Reply:
There is 'close' class because the 'save' button is also closing the popup

Reply:
It is better to call class _close_add_family_popup_